cmd/yes/yes.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+func parseArguments() []string {
+	flag.Parse()
+
+	strings := flag.Args()
+
+	return strings
+}
+
+func main() {
+	strings := parseArguments()
+
+	if len(strings) != 0 {
+		for {
+			for _, string := range strings {
+				fmt.Print(string, " ")
+			}
```

Comment:
use `strings.Join` here too

Reply:
also why not defining the string to be printed only once before the loop(can be `y` or the user input) then print it infinitely, no need to process the args all time